File: Problem 1/problem1-BruteForce.py
# Brute Force approach: Loop from 1 to the last index and check divisibility. If
# the number is divisible by 5 or 3, print the output

sum_multiples = 0
for i in range(1, 1000):
  if (i % 3 == 0 or i % 5 == 0):
    sum_multiples += i

print(sum_multiples)

File: Problem 1/problem1BruteForce.java
class problem1BruteForce{
  public static void main(String[] args) {
    long summultiples = 0;
    for (int i = 0; i < 1000; i++) {
      if(i % 3 == 0 || i % 5 == 0) {
        summultiples += i;
      }
      else {
        continue;
      }
    }

    System.out.println(summultiples);
  }
}